projects/plugins/jetpack/extensions/extended-blocks/paid-blocks/with-upgrade-banner.jsx

```diff
@@ -41,8 +43,9 @@ export default createHigherOrderComponent(
 				plan: requiredPlan,
 				blockName: name,
 				context: bannerContext,
+				tracks,
 			} ),
-			[ requiredPlan, name, bannerContext ]
```

Comment:
There was no need for `bannerContext` here, the variable cannot change.

Reply:
`react-hooks/exhaustive-deps` might've caught this (and any missing deps elsewhere)

https://reactjs.org/docs/hooks-rules.html
https://github.com/facebook/react/tree/main/packages/eslint-plugin-react-hooks